config/routes.rb

```diff
@@ -16,4 +20,16 @@
     resources :comments, only: [:new, :create, :destroy]
     resources :likes, only: :create
   end
+
+  namespace :api do 
+    namespace :v1 do
+      resources :users, only: [index] do
```

Comment:
- [ ] I'm unable to setup your the project locally because of the error on the screenshot below:


![Screenshot from 2022-10-01 17-43-41](https://user-images.githubusercontent.com/63087491/193419551-2fd46d77-0531-4e2b-8b7e-8134e5743565.png)

Please kindly fix it.